litmus-portal/frontend/src/components/VerifyCommit/index.tsx

```diff
@@ -94,7 +94,7 @@ function VerifyCommit() {
                   <ButtonOutline
                     isDisabled={false}
                     handleClick={() => {
-                      console.log('Edit');
+                      // console.log('Edit');
```

Comment:
remove the code comments in all the places

Reply:
This is not my code, it was giving eslint warnings so I commented it.

Reply:
Is it possible to remove it?

Reply:
Not sure, they might have kept it for some reason.